src/SrvCommand.cpp

```diff
@@ -55,8 +50,8 @@ void SPModInfoCommand()
                                        std::setw(authWidth),
                                       "author",
                                       "filename");
-            std::size_t pos = 1;
-            for (auto entry : gSPGlobal->getPluginManagerCore()->getPluginsList())
+            //std::size_t pos = 1;
```

Comment:
Rewrite this for exts?

Reply:
Forgot about this

Reply:
Ready, but need to wait until https://github.com/dreamstalker/rehlds/pull/697 gets merged.

Reply:
RP merged

Reply:
It's fixed in the current revision